setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -55,15 +55,16 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $errors = $this->validate($input);
+        $data = $this->getUserData($input, $output);
+        $errors = $this->validate($data);
         if ($errors) {
             $output->writeln('<error>' . implode('</error>' . PHP_EOL .  '<error>', $errors) . '</error>');
             return;
         }
         $installer = $this->installerFactory->create(new ConsoleLogger($output));
-        $installer->installAdminUser($input->getOptions());
+        $installer->installAdminUser($data);
         $output->writeln(
-            '<info>Created Magento administrator user named ' . $input->getOption(AdminAccount::KEY_USER) . '</info>'
+            '<info>Created Magento administrator user named ' . $data[AdminAccount::KEY_USER] . '</info>'
```

Comment:
`getOption()` verified that such option is supported and that it is set. Accessing it directly in the array can be risky because there is no guarantee that such key exists. I'd suggest to add some validation for all such cases. Maybe a new private method can be dedicated to it.